### PR TITLE
New Template for G Suite + HTTP

### PR DESCRIPTION
**Pull request by arvilnagpal-okta**
@charleschan-okta @davidroossien-okta Could you guys please review and approve?

@richardheng-okta fyi

### CODE REVIEW

File: workflows/raw_http_connection_gsuite-main/workflow.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+   "name": "raw_http_connection_gsuite",
```
Comment (charleschan-okta) [inline workflows/raw_http_connection_gsuite-main/workflow.json:2]: The folder name is: raw_http_connection_gsuite-main, which does not conform to the rule:
1. Folder name needs to be lower case and separated with underscore

Do u wanna use: "raw_http_connection_gsuite-main" or " raw_http_connection_gsuite"?
If u use the latter, the folder name and workflow name would match then

File: workflows/raw_http_connection_gsuite-main/workflow.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+   "title": "Connect to Any G Suite API with the HTTP Connector",
+   "description": "Our current Workflows connector for Gsuite Admin does not include all functions within Google Workspace. The Custom API card is also restricted to the Directory and Licensing APIs. If there are other actions that you’d like to execute the Google has exposed an API for, we can use this guide to create a raw HTTP connector and use that to request the scopes needed.",
+   "connectors": [
+      "okta"
```
Comment (charleschan-okta) [inline workflows/raw_http_connection_gsuite-main/workflow.json:6]: There is no "okta" connector used in the flow

File: workflows/raw_http_connection_gsuite-main/workflow.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+   ],
+   "links": [
+      {
+         "title": "Video Walk-through",
+         "destination": "https://www.youtube.com/watch?v=y00PYCgnQfI"
+      },
+      {
+         "title": "Setup Documentation",
+         "destination": "https://github.com/okta/workflows-templates/blob/master/workflows/raw_http_connection_gsuite/readme.md"
```
Comment (charleschan-okta) [inline workflows/raw_http_connection_gsuite-main/workflow.json:15]: Pl make sure the path here match the folder path

**Review by charleschan-okta — CHANGES_REQUESTED** [review]
Some issues w/ the readme and workflow.json files

File: workflows/raw_http_connection_gsuite-main/readme.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+
+*   Access to an Okta tenant with Okta Workflows enabled for your org
+*   Access to a tenant for Google Workspace
+
+## Workflow Setup Steps
+
+
+
+1. Navigate to[ https://console.developers.google.com/](https://console.developers.google.com/) and login with the G Suite Admin login.
```
[the remark this answers is not in the document]
Comment (sscode52) [reply workflows/raw_http_connection_gsuite-main/readme.md:23]: You would need Super Admin permissions in Gsuite to get the required scopes and run the flows.

File: workflows/raw_http_connection_gsuite-main/workflow.json (hunk 3 above)
Comment (sscode52) [reply workflows/raw_http_connection_gsuite-main/workflow.json:15]: This should match in the new PR #58

**Comment by charleschan-okta** [comment]
Arvil created this PR but it’s superceded by PR#58 by Sowmya. So close this PR

**Pull request closed without merging**